git_machete/client.py

```diff
@@ -636,6 +636,7 @@ def traverse(
         self.expect_at_least_one_managed_branch()
 
         self.__empty_line_status = True
+        is_action_done: bool = False
```

Comment:
`any_action_done`? also, this name is not adequate... rather `any_action_suggested` (?)